apps/web-giddh/src/app/inventory/components/group-stock-report-component/group.stockreport.component.html

```diff
@@ -93,7 +211,8 @@
     </div>
 
     <section>
-        <table class="table basic table-border inventoryStockTable"
+        <div style="overflow-x:auto" class="mobilecardwrapper table-responsive">
```

Comment:
remove inline css, the class name should be in hyphen-case

Reply:
done